packages/shared/src/components/CharacterStats/CharactersOverview.tsx

```diff
@@ -0,0 +1,199 @@
+import _ from 'lodash';
+import { useMemo } from 'react';
+import {
+  LabelList,
+  Legend,
+  PolarAngleAxis,
+  PolarGrid,
+  PolarRadiusAxis,
+  Radar,
+  RadarChart,
+  ResponsiveContainer,
+  Tooltip,
+} from 'recharts';
+import { Props } from 'recharts/types/component/Label';
+
+import { UserCharacterInfo } from '../../graphql/__generated__/graphql';
+import { Utils } from '../../utils/utils';
+
+const ClassLabel = (props: {
+  cx: number;
+  cy: number;
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+  radius: number;
+  textAnchor: string;
+  payload: { value: string };
+}) => {
+  const classId = parseInt(props.payload.value) as CombatUnitClass;
+  const className =
+    classId === CombatUnitClass.DeathKnight
+      ? 'DK'
+      : classId === CombatUnitClass.DemonHunter
+      ? 'DH'
+      : Utils.getClassName(classId);
+  return (
+    <g className="recharts-layer recharts-polar-angle-axis-tick">
+      <text
+        className="recharts-text recharts-polar-angle-axis-tick-value"
+        fill={getClassColor(classId)}
+        type="category"
+        cx={props.cx}
+        cy={props.cy}
+        orientation="outer"
+        radius={props.radius}
+        x={props.x}
+        y={props.y}
+        textAnchor={props.textAnchor}
+        fontWeight="bold"
+      >
+        <tspan x={props.x} dy="0em">
+          {className}
+        </tspan>
+      </text>
+    </g>
+  );
+};
+
+const RatingLabel = (props: Props) => {
+  if (props.value === 0) return null;
+  return (
+    <g className="recharts-layer recharts-polar-angle-axis-tick">
+      <text
+        className="recharts-text recharts-polar-angle-axis-tick-value"
+        fill="#fff"
+        type="category"
+        cx={props.cx}
+        cy={props.cy}
+        orientation="outer"
+        radius={props.radius}
+        x={props.x}
+        y={props.y}
+        textAnchor={props.textAnchor}
+      >
+        <tspan x={props.x} dy="0em">
+          {props.value}
+        </tspan>
+      </text>
+    </g>
+  );
+};
+
+export const CharactersOverview = (props: { characters: UserCharacterInfo[] }) => {
+  const latestRatingPerClass = useMemo(() => {
+    const data = _.map(
+      _.groupBy(
+        props.characters.flatMap((c) =>
+          c.bracketStats.map((s) => ({ ...s, classId: Utils.getSpecClass(c.specId as CombatUnitSpec) })),
+        ),
+        'classId',
+      ),
+      (bracketStats, classId) => {
+        return {
+          classId,
+          rating2v2:
+            _.max(
+              _.map(
+                bracketStats.filter((s) => s.bracket === '2v2'),
+                (s) => s.latestRating,
+              ),
+            ) ?? 0,
+          rating3v3:
+            _.max(
+              _.map(
+                bracketStats.filter((s) => s.bracket === '3v3'),
+                (s) => s.latestRating,
+              ),
+            ) ?? 0,
+          ratingShuffle:
```

Comment:
Isn't rating in shuffle by spec?

Reply:
I suppose we are just using highest of all specs the class could be? Might be worth extending this at some point to show all of them

Reply:
right, currently it's class based. doing it at spec level will make the radar chart very dense. need to think about how to present it.